File: ezSCORES/ezSCORES.Model/CompetitionsTeams.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace ezSCORES.Model
{
	public class CompetitionsTeams
	{
		public int Id { get; set; }

		public int CompetitionId { get; set; }

		public int TeamId { get; set; }

		public int? GroupId { get; set; }

		public bool IsEliminated { get; set; }
		public virtual ICollection<CompetitionsTeamsPlayers> CompetitionsTeamsPlayers { get; set; } = new List<CompetitionsTeamsPlayers>();
	}
}

File: ezSCORES/UI/ezscores_mobile/lib/models/competitionsTeams.dart
import 'package:ezscores_mobile/models/competitionsTeamsPlayers.dart';
import 'package:ezscores_mobile/models/teams.dart';
import 'package:json_annotation/json_annotation.dart';

part 'competitionsTeams.g.dart';

@JsonSerializable()
class CompetitionsTeams {
	int? id;
	int? competitionId;
	int? teamId;
	int? groupId;
	bool? isEliminated;
  Teams? team;
	List<CompetitionsTeamsPlayers>? competitionsTeamsPlayers;

	CompetitionsTeams();

	factory CompetitionsTeams.fromJson(Map<String, dynamic> json) => _$CompetitionsTeamsFromJson(json);

	Map<String, dynamic> toJson() => _$CompetitionsTeamsToJson(this);
}
